platform/linux-generic/odp_pool.c

```diff
@@ -1856,8 +1856,21 @@ int odp_pool_ext_capability(odp_pool_type_t type, odp_pool_ext_capability_t *cap
 {
 	odp_pool_stats_opt_t supported_stats;
 
-	if (type != ODP_POOL_PACKET)
+	_ODP_ASSERT(capa != NULL);
+
+	switch (type) {
+	case ODP_POOL_PACKET:
+		break;
+	case ODP_POOL_BUFFER:
+	case ODP_POOL_TIMEOUT:
+	case ODP_POOL_VECTOR:
+	case ODP_POOL_DMA_COMPL:
+		capa->max_pools = 0;
```

Comment:
It would be safer to memset(0) also in this case. If application would miss the max_pools check, it would bump into all other fields being zero (instead of uninitialized).

Reply:
Changed in V4.